constants.go

```diff
@@ -86,8 +86,8 @@ const (
 
 // Constants for GitHub credential secret.
 const (
-	// CredentialSecretName is a secret name for GitHub credential secret.
-	CredentialSecretName = "github-cred"
+	// DefaultCredentialSecretName is the default secret name for GitHub credential secret.
+	DefaultCredentialSecretName = "github-cred"
```

Comment:
I thought it would be better to add the `meows-`prefix to make it clear that `DefaultCredentialSecretName` is used by meows.
However, this point may not be particularly meaningful, since the `.spec.credentialSecretName` of a resource can be optional specified by the user.